src/components/common/PageHeader/index.tsx

```diff
@@ -1,28 +1,26 @@
 import { Box, Typography } from '@mui/material'
-import type { BoxProps } from '@mui/material'
 import type { ReactElement } from 'react'
 
 import css from './styles.module.css'
 
 const PageHeader = ({
   title,
   action,
-  sx = {},
+  noBorder,
 }: {
   title: string
   action?: ReactElement
-  sx?: BoxProps['sx']
+  noBorder?: boolean
 }): ReactElement => {
   return (
     <Box
       className={css.container}
       sx={{
-        height: '88px',
         top: '-12px',
-        ...sx,
+        borderBottom: !noBorder ? ({ palette }) => `1px solid ${palette.border.light}` : undefined,
```

Comment:
Splitting hairs but this would read better:

```suggestion
        borderBottom: noBorder ? undefined : ({ palette }) => `1px solid ${palette.border.light}`,
```